### DISCUSSION TITLE
EAGE HACK 2023: I have an idea to hack on!

### DISCUSSION THREAD

**Discussion by aruss175** [EAGE-Annual-Hackathon/EAGE-Annual-Hackathon-Discussion-Boards#9] [Ideas]
Please use this thread to post any ideas you have around natural language processing for geoscience or engineering.  These are the ideas that teams should form around.

**Comment by aruss175** [comment]
**The EAGE has generously made available ALL the 2023 conference abstracts which is a huge dataset for this hackathon**.  There are a lot of ideas that you can do with this set of data.  But first, you will need to do some prep - OCR, maybe some labelling - and then once you are in all text, time to apply a language model over top.  There comes the challenge - the language in this dataset is specific to quite technical depth in the geosciences and engineering - which language model do you choose?  Then how to create a way to interact with it?

Another idea is to look for patterns in this text dataset - what are the trends in the EAGE 2023 abstracts?  What about additional metadata with the abstracts (authors, companies...etc) - does this tell us anything about the state of our science?

**Reply by biscomusic** [reply]
Do we know what format will this data be provided in? Assume pdfs in a zip?

**Comment by gpap-gpap** [comment]
I am keen to try out the idea for using the dataset to understand trends - I also have a notebook that crawls google scholar for past EAGE abstracts - this needs to have a mini pause so that it conforms with google's bot policy so might need to run overnight and even then google might ban requests from it but whatever I get from it could be used in conjunction with what's on offer on EAGE.

**Reply by biscomusic** [reply]
I had similar thoughts about trends… along the lines of who is submitting what types and kinds of content
